frontend/public/components/volumes-table.tsx

```diff
@@ -5,44 +5,58 @@ import * as classNames from 'classnames';
 
 import {
   ContainerSpec,
+  getVolumeType,
+  getVolumeLocation,
+  K8sResourceKind,
   PodKind,
   PodTemplate,
   Volume,
   VolumeMount,
 } from '../module/k8s';
-import {
-  getVolumeType,
-  getVolumeLocation,
-} from '../module/k8s/pods';
-import {
-  VolumeIcon,
-  ResourceIcon,
-  EmptyBox,
-  SectionHeading,
-} from './utils';
+import { VolumeIcon, ResourceIcon, EmptyBox, SectionHeading, VolumeKebab, Kebab, asAccessReview, VolumeKebabFactory } from './utils';
+import { Table, TableData, TableRow } from './factory';
+import { sortable } from '@patternfly/react-table';
+import { removeVolumeModal } from './modals';
 
-const getVolumeMountsByPermissions = (pod: PodTemplate): VolumeData[] => {
+const VolumeKebabMenu: VolumeKebabFactory = {
```

Comment:
Having a factory here is overkill since we only have one action. Just define the single action as a `const` variable